Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/describe.go

```diff
@@ -602,6 +603,9 @@ func describePod(pod *api.Pod, events *api.EventList) (string, error) {
 	return tabbedString(func(out io.Writer) error {
 		fmt.Fprintf(out, "Name:\t%s\n", pod.Name)
 		fmt.Fprintf(out, "Namespace:\t%s\n", pod.Namespace)
+		if pod.Annotations[allocator.ValidatedSCCAnnotation] != "" {
```

Comment:
@smarterclayton - sorry about that, I didn't pick up on that convention, so good to know.  I've learned a lot today :)   
OK, changed to "Security Policy" which I think will make more sense for current SCC and future PSP in upstream.  @pweil- @kargakis - you cool with that?